modules/36_buildings/simple/datainput.gms

```diff
@@ -35,15 +35,25 @@ pm_cesdata_sigma(ttot,"enhgab")$ (ttot.val eq 2040) = 3;
 
 
 *** floor space demand for reporting
+* RH, Jan 2023: if clauses to maintain backwards compatibility with old input data for a while
```

Comment:
If people want to know who commented what when, they can use `git blame`.  My opinion.

Reply:
I agree, this information will not be helpful in 2 years as also it might not be clear, who "RH" is :-)